genesyscloud/resource_genesyscloud_outbound_contactlist_test.go

```diff
@@ -0,0 +1,214 @@
+					),
+					generatePhoneColumnsBlock(
+						"Home",
+						"home",
+						"Home",
+					),
+				),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("genesyscloud_outbound_contact_list."+resourceId, "name", name),
+					resource.TestCheckResourceAttr("genesyscloud_outbound_contact_list."+resourceId, "column_names.0", "Cell"),
+					resource.TestCheckResourceAttr("genesyscloud_outbound_contact_list."+resourceId, "column_names.1", "Home"),
+					resource.TestCheckResourceAttr("genesyscloud_outbound_contact_list."+resourceId, "phone_columns.0.column_name", "Cell"),
+					resource.TestCheckResourceAttr("genesyscloud_outbound_contact_list."+resourceId, "phone_columns.0.type", "cell"),
+					resource.TestCheckResourceAttr("genesyscloud_outbound_contact_list."+resourceId, "phone_columns.0.callable_time_column", "Cell"),
+					resource.TestCheckResourceAttr("genesyscloud_outbound_contact_list."+resourceId, "phone_columns.1.column_name", "Home"),
+					resource.TestCheckResourceAttr("genesyscloud_outbound_contact_list."+resourceId, "phone_columns.1.type", "home"),
+					resource.TestCheckResourceAttr("genesyscloud_outbound_contact_list."+resourceId, "phone_columns.1.callable_time_column", "Home"),
+					resource.TestCheckResourceAttr("genesyscloud_outbound_contact_list."+resourceId, "preview_mode_column_name", previewModeColumnName),
+					resource.TestCheckResourceAttr("genesyscloud_outbound_contact_list."+resourceId, "preview_mode_accepted_values.0", previewModeColumnName),
+					resource.TestCheckResourceAttr("genesyscloud_outbound_contact_list."+resourceId, "automatic_time_zone_mapping", automaticTimeZoneMapping),
+					testDefaultHomeDivision("genesyscloud_outbound_contact_list."+resourceId),
+				),
+			},
+			{
+				Config: generateAttemptLimitResource(
+					attemptLimitResourceID,
+					attemptLimitName,
+					"5",
+					"5",
+					"America/Chicago",
+					"TODAY",
+				) + generateOutboundAttemptLimitDataSource(
+					attemptLimitDataSourceID,
+					attemptLimitName,
+					"genesyscloud_outbound_attempt_limit."+attemptLimitResourceID,
+				) + `data "genesyscloud_auth_division_home" "home" {}` + generateOutboundContactList(
+					resourceId,
+					nameUpdated,
+					"data.genesyscloud_auth_division_home.home.id",
+					previewModeColumnNameUpdated,
+					previewModeAcceptedValuesUpdated,
+					columnNamesUpdated,
+					automaticTimeZoneMappingUpdated,
+					zipCodeColumnName,
+					"genesyscloud_outbound_attempt_limit."+attemptLimitResourceID+".id",
+					generatePhoneColumnsBlock(
+						"Cell",
+						"cell",
+						"",
+					),
+					generatePhoneColumnsBlock(
+						"Home",
+						"home",
+						"",
+					),
+				),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("genesyscloud_outbound_contact_list."+resourceId, "name", nameUpdated),
+					resource.TestCheckResourceAttr("genesyscloud_outbound_contact_list."+resourceId, "division_id", "4a7e4f4b-e734-40b6-838e-557c5eedd49a"),
```

Comment:
Looks like you'll need to update this check also

Reply:
Done. Removed it since the `testDefaultHomeDivision` function is doing that underneath